src/layout/organize/JourneyInstanceLayout.tsx

```diff
@@ -76,14 +79,16 @@ const JourneyInstanceLayout: React.FunctionComponent = ({ children }) => {
                 style={{ marginRight: '0.25rem' }}
               />
               <Typography>
-                {journeyInstance.next_milestone?.title}
+                {journeyInstance.next_milestone.title}
                 {': '}
-                <FormattedDate
-                  day="numeric"
-                  month="long"
-                  value={journeyInstance.next_milestone?.deadline}
-                  year="numeric"
-                />
+                {journeyInstance.next_milestone.deadline && (
+                  <FormattedDate
+                    day="numeric"
+                    month="long"
+                    value={journeyInstance.next_milestone.deadline}
+                    year="numeric"
+                  />
+                )}
```

Comment:
This colon should be inside the conditional right?

Reply:
Ah yeah I simply forgot about this. If this is the only issue I agree it can wait, but it was a good catch!